catalog/imaging/imaging-transformer-nitf/pom.xml

```diff
@@ -109,6 +109,13 @@
             <artifactId>jai-imageio-jpeg2000</artifactId>
             <version>${jpeg2000.version}</version>
         </dependency>
+
+        <dependency>
```

Comment:
Ugh. Can't believe I missed this last night. This should be using the new shaded jar:
```xml
        <!--Spock dependencies-->
        <dependency>
            <groupId>ddf.lib</groupId>
            <artifactId>spock-shaded</artifactId>
            <version>${ddf.version}</version>
            <scope>test</scope>
        </dependency>
```